db/migrate/20200108180542_create_sample_types.rb

```diff
@@ -27,6 +27,7 @@ def populate_sample_types!
       'CNS Infections': [
         { name: 'Brain' },
         { name: 'Cerebrospinal Fluid', human_only: true },
+        { name: 'Central Nervous System' },
         { name: 'Ocular Fluid', human_only: true },
       ],
 
```

Comment:
May make mores sense (for consistency across the sample groups) to use 
- "Central Nervous System" instead of "CNS Infections"
- "Respiratory Tract" instead of "Respiratory Infections" 
- "Reproductive Tract" instead of "Reproductive"